lti_consumer/api.py

```diff
@@ -53,10 +59,22 @@ def _get_lti_config(config_id=None, block=None):
     if config_id:
         lti_config = LtiConfiguration.objects.get(pk=config_id)
     elif block:
-        lti_config = _get_or_create_local_lti_config(
-            block.lti_version,
-            block.location,
-        )
+        if block.config_type == 'external':
+            config = get_external_config_from_filter(
+                {"course_key": block.location.course_key},
+                block.external_config
+            )
+            lti_config = _get_or_create_local_lti_config(
+                config.get("version"),
+                block.location,
+                block.external_config
+            )
+            lti_config.lti_config = config
```

Comment:
@tecoholic Why is this line? We are always using the filter pipeline to retrieve the configuration, so this can become outdated when the credentials are updated in the LTI Store plugin.

Reply:
I agree. We don't need this.